src/SFML/Window/iOS/SFAppDelegate.mm

```diff
@@ -151,6 +162,36 @@ - (void)applicationWillTerminate:(UIApplication *)application
     }
 }
 
+- (bool)supportsOrientation:(UIDeviceOrientation)orientation
+{
+    if (!self.sfWindow)
```

Comment:
Yes. Don't hesitate to add empty lines in your code to make it more redable, by separating logical groups.